packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -1114,6 +1117,60 @@ function useDeleteAiChat() {
   );
 }
 
+/**
+ * Returns a function to send a message in an AI chat.
+ *
+ * @example
+ * const sendMessage = useSendAiMessage(chatId);
+ * sendMessage("Hello, Liveblocks AI!");
+ */
```

Comment:
@nimeshnayaju FYI, I just went over this for the rest of the hooks, and fixed the package linting issues that reported some hooks were missing doc strings, so while I was in that mode, I also added it here for your newly added hook 👍